.github/workflows/ci.yml

```
@@ -62,33 +62,28 @@ jobs:
       with:
         submodules: true
     - name: Prepare base image in Docker
-      run: docker build . -f utils/docker/Dockerfile.centos.8
+      run: docker build . -f utils/docker/Dockerfile.fodera
```

Spelling mistake.  You'll need to modify this in lots of places and rename the file.
```suggestion
      run: docker build . -f utils/docker/Dockerfile.fedora
```
Although now fedora is in it's own file you can collapse this a bit more, add ", fedora" to the matix azis on L18 and remove Lines 51 and onwards in this file.